src/components/gameRulesList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import '../App.css';
```

Comment:
There's no need to import App.css here as it is the global CSS. component styles should be put in the in the component's counterpart CSS file. In this case 'gameRulesList.css' and then imported into the component.
```suggestion
import './gameRulesList.css';
```